tox.ini

```diff
@@ -66,9 +67,11 @@ deps =
     flake8
     readme-renderer
     pygments
+    isort
```

Comment:
Imho we should copy the pytest core linting   https://github.com/pytest-dev/pytest/blob/master/tox.ini#L52

Reply:
That's a good idea for after this pull request.